docs/pages/develop/testing.mdx

```diff
@@ -0,0 +1,319 @@
+---
+title: Testing with Jest
+sidebar_title: Testing
+description: Learn how to set up and configure the jest-expo library to write unit and snapshot tests for a project.
```

Comment:
```suggestion
description: Learn how to set up and configure the jest-expo library to write unit and snapshot tests for a project with Jest.
```
Maybe too redundant, was just thinking about how it's specifically about Jest, and how there's other ways to write unit tests